resources/views/settings/subscription/index.blade.php

```diff
@@ -6,6 +6,17 @@
  */
 
 ?>
+@php
```

Comment:
This should be on the _pricing_ page, not the _subscription_ page, where we already have a currency changer